src/test/modules/test_extensions/expected/test_extensions.out

```diff
@@ -73,12 +80,9 @@ SELECT point(0,0) <<@@ polygon(circle(point(0,0),1));
 (1 row)
 
 \dx+ test_ext_cor
-Objects in extension "test_ext_cor"
-      Object Description      
-------------------------------
- function ext_cor_func()
- operator <<@@(point,polygon)
- type test_ext_type
- view ext_cor_view
-(4 rows)
+                       List of installed extensions
+     Name     | Version | Schema |              Description               
+--------------+---------+--------+----------------------------------------
+ test_ext_cor | 1.0     | public | Test extension using CREATE OR REPLACE
+(1 row)
```

Comment:
seems not all test cases exists in this file

Reply:
True. Behavior should be the same. 
I think we can adopt whole test file as well.

Reply:
Test file was adopted